cloudwatch_alarms/main.tf

```diff
@@ -0,0 +1,16 @@
+resource "aws_cloudwatch_metric_alarm" "cloudwatch_metric_alarm" {
+  alarm_name          = "${var.environment}-${var.queue_name}-DLQ-alarm"
```

Comment:
It's likely this module will be used for more than just DLQ alarms, maybe make the `name` a variable to pass in so teams can set it to something meaningful based on context.